pkg/oc/cli/admin/router/router.go

```diff
@@ -780,7 +780,10 @@ func RunCmdRouter(f kcmdutil.Factory, cmd *cobra.Command, out, errout io.Writer,
 		"STATS_PORT":                            strconv.Itoa(cfg.StatsPort),
 		"STATS_USERNAME":                        cfg.StatsUsername,
 		"STATS_PASSWORD":                        cfg.StatsPassword,
-		"ROUTER_THREADS":                        strconv.Itoa(int(cfg.Threads)),
+	}
+
+	if cfg.Threads > 0 {
+		env["ROUTER_THREADS"] = strconv.Itoa(int(cfg.Threads))
```

Comment:
A user could enter `--threads=4000000000`, and we would end up setting `ROUTER_THREADS` to -294967296.  It would be more correct to avoid using a signed integer for the intermediate value:
```go
		env["ROUTER_THREADS"] = strconv.FormatUint(uint64(cfg.Threads), 10)
```